src/component/Category.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import NavBar from './NavBar';
```

Comment:
Has 2 `<Link>`s in the navbar which leads to the booklist and categories locations 🙌